tortoise/backends/base/schema_generator.py

```diff
@@ -241,6 +279,10 @@ def _get_field_sql_and_related_table(
                 to_field_name = reference.to_field_instance.model_field_name
 
             related_table_name = reference.related_model._meta.db_table
+            # Get qualified table name for FK reference if related model has schema
+            qualified_related_table_name = self._get_qualified_table_name(
+                reference.related_model
+            ).strip('"')
```

Comment:
Why would you remove the leading and trailing `"` but leave them in the middle around `.`?